### DISCUSSION TITLE
Who can I ask for a personal signup?

### DISCUSSION THREAD

**Discussion by WeberAlex** [thedevs-network/kutt#671] [Q&A]
Hi,

I would like to sign up. I read the reply about the SPAM problem. I would like to use Kutt for my personal usage. No SPAM, no Commercial usage. Juste a lightway usage of my private links through ShareX. Is this possible? Thanks.

**Comment by MichaelTunnell** [comment]
Kutt is not a hosted service, this a software project that you host yourself. This is not that clear on the homepage of the project but if you want to host it yourself you need to have your own server or using a cloud provider like [DigitalOcean](https://do.co/tux2022)
